scripts/qa_crowbarsetup.sh

```diff
@@ -1475,6 +1475,41 @@ function onadmin_repocleanup()
     zypper mr -d sp3sdk
 }
 
+# replace zypper repos from the image with user-specified ones
+# because clouddata might not be reachable from where this runs
+function onadmin_reporeplace()
```

Comment:
Or even setup_local_zypper_repositories. Since it is only called twice, using a shortcut instead of the full name does more harm than good IMHO.